include/d/d_demo.h

```diff
@@ -24,13 +24,7 @@
 #include "f_pc/f_pc_base.h"
 
 class fopAc_ac_c;
-
-class dMesg_tControl : public JMessage::TControl {
-public:
-    dMesg_tControl();
-
-    /* 0x3C */ u8 field_0x3C[0x74 - 0x3C];
```

Comment:
you have to move the members here or else the size (0x74) will be wrong, breaking the match in dDemo_manager_c